version.go

```diff
@@ -21,10 +21,10 @@ type Version struct {
 	Description     string `json:"description,omitempty" structs:"description,omitempty"`
 	Archived        bool   `json:"archived,omitempty" structs:"archived,omitempty"`
 	Released        bool   `json:"released,omitempty" structs:"released,omitempty"`
-	ReleaseDate     string `json:"releaseDate,omitempty" structs:"releaseDate,omitempty"`
+	ReleaseDate     *Date  `json:"releaseDate,omitempty" structs:"releaseDate,omitempty"`
 	UserReleaseDate string `json:"userReleaseDate,omitempty" structs:"userReleaseDate,omitempty"`
```

Comment:
Why not this one?

Reply:
We don't use it anywhere in our code base and it's a different timeformat `"userReleaseDate": "6/Jul/2010",` so I'd have to add support for it. I can do that if you want, otherwise leaving it as a string won't cause unmarshalling errors.